l10n/en/footer-refresh.ftl

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+footer-refresh-find-out-about = Find out about { -brand-name-mozilla } products, initiatives, and more. We’ll never sell your email, even if presented with free pizza for life.
+footer-refresh-leadership = Leadership
+footer-refresh-advertise = Advertise with { -brand-name-mozilla }
+footer-refresh-firefox-release-notes = { -brand-name-firefox } Release Notes
+footer-refresh-mdn = MDN
+footer-refresh-follow-mozilla = Follow @{ -brand-name-mozilla }
+footer-refresh-x = X
+footer-refresh-x-formerly-twitter = X (formerly Twitter)
+footer-refresh-mastodon = { -brand-name-mastodon }
+footer-refresh-instagram = { -brand-name-instagram }
+footer-refresh-linkedin = { -brand-name-linkedin }
+footer-refresh-tiktok = { -brand-name-tiktok }
+footer-refresh-spotify = { -brand-name-spotify }
```

Comment:
**Nit:** we've moved away from using third party brand name variables, so you can use regular strings here instead. Only Mozilla's own products and brand names should use these types of variable in new files like this.